doc/README.md

```diff
@@ -131,7 +131,7 @@ The required variables are as follows:
 | SECRET_KEY_BASE            | 65 chars long random string used to sign cookies |
 | GITHUB_CLIENT_ID           | OAuth Client ID of your GitHub App               |
 | GITHUB_CLIENT_SECRET       | OAuth Client Secret of your GitHub App           |
-| GITHUB_INTEGRATION_ID      | The ID of the installation created before        |
+| GITHUB_APP_ID              | The ID of the installation created before        |
```

Comment:
Since you are changing this already, could you change the left value to `The ID of the App created before`?